test/ZNSRootRegistrar.test.ts

```diff
@@ -103,6 +103,73 @@ describe("ZNSRootRegistrar", () => {
     await mongoAdapter.dropDB();
   });
 
+  it("Should register an array of domains", async () => {
+    const registrations = [];
+
+    for (let i = 0; i < 5; i++) {
+      const isOdd = i % 2 !== 0;
+
+      const domainObj = {

```

Comment:
create a type for this object if we do not have one already so that in the future should we need to change anything or redo similar work, it shows an error here when the type itself is modified